Lib/test/test_platform.py

```diff
@@ -328,6 +328,27 @@ def test_java_ver(self):
 
     def test_win32_ver(self):
         res = platform.win32_ver()
+        self.assertEqual(len(res), 4)
+        if support.MS_WINDOWS:
+            release, version, csd, ptype = res
+            if release:
+                releases = []
+                releases.extend(r for _k, r in platform._WIN32_CLIENT_RELEASES)
+                releases.extend(r for _k, r in platform._WIN32_SERVER_RELEASES)
+                self.assertIn(release, releases)
```

Comment:
This is testing our current implementation, but it isn't the documented behaviour of the function. We might decide to start returning the OS's own reported name, which would be fine without any deprecation period.

Worth adding a comment here `# Currently, release names always come from these internal dicts, but this could change over time`. Alternatively, just check that `release` is something other than what was originally passed to `win32_ver`.

Reply:
Yes, I will probably go with the second option 👍